src/data.js

```diff
@@ -19,6 +19,9 @@ var Hook = Entity.configure({
     task:               Entity.types.JSON,
     // pulse bindings (TODO; empty for now)
     bindings:           Entity.types.JSON,
+    // timings for the task (in fromNow format, e.g., "1 day")
+    deadline:           Entity.types.String,
+    expires:            Entity.types.String,
```

Comment:
whoops, I don't think you meant to add this here!

Reply:
I thought you asked to keep the older versions intact so I added them back there. Sorry for misunderstanding. Correcting it.